test/e2e/helm/e2e_suite_test.go

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
Could we have a PR only for the e2e tests?
Ideally we should have one PR for each purpose 
AND 
with only one commit

Could you please do that for we are able to move forward?

Reply:
regarding a separate PR for e2e-tests:
let us please came up with an idea on https://github.com/kubernetes-sigs/kubebuilder/pull/4903#discussion_r2185687198 and https://github.com/kubernetes-sigs/kubebuilder/pull/4903#discussion_r2185686431 first :pray: 

regarding squashing: let's decided on :point_up: and than I can squash all the commits which belong to this PR.

Reply:
just raised PR #4914 for testing the existing behavior of `kubebuilder edit --plugins helm.kubebuilder.io/v1-alpha`

Reply:
rebased and pushed again - PTAL :pray: